Cargo.lock

```diff
@@ -9109,6 +9109,36 @@ name = "opendal"
 version = "0.44.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4af824652d4d2ffabf606d337a071677ae621b05622adf35df9562f69d9b4498"
+dependencies = [
+ "anyhow",
+ "async-trait",
+ "backon",
+ "base64 0.21.0",
+ "bytes",
+ "chrono",
+ "flagset",
+ "futures",
+ "getrandom 0.2.11",
+ "http",
+ "log",
+ "md-5",
+ "once_cell",
+ "percent-encoding",
+ "prometheus",
+ "quick-xml 0.30.0",
+ "reqsign",
+ "reqwest",
+ "serde",
+ "serde_json",
+ "sha2",
+ "tokio",
+ "uuid",
+]
+
+[[package]]
+name = "opendal"
+version = "0.44.2"
+source = "git+https://github.com/apache/opendal.git?rev=e94f068#e94f068a9977fcd20c9bb3626aaaa8be833209a8"
```

Comment:
Seems we are refering to two version of opendal. Please use `repalce` instead change `opendal` directly.